usefulUtilities/statusIndicator/webApp/app/statusIndicator_webApp.js

```diff
@@ -328,46 +254,33 @@ function handleCanaryRequest(res) {
         },
         (error, response) => {
             allEmployeesPageOK = !error;
-            
-            request.get(
-                config.wwwRoot + "teamPage.html",
-                {
-                    timeout: 15000
-                },
-                (error, response) => {
-                    teamPageOK = !error;
-
-                    var query = "SELECT 1;"
-                    connection.query(query, function (error, results, fields) {
-                        sqlConnectionOK = !error;
-
-                        // The `result` value should be a logical AND
-                        // of all of the subsystems whose status we check.
-                        var responseObject = {
-                            result: apiRouterOK && allEmployeesPageOK && teamPageOK && sqlConnectionOK,
-                            systemStatus: {
-                                apiRouter: {
-                                    status: apiRouterOK
-                                },
-                                http: {
-                                    allEmployeesPage: {
-                                        status: allEmployeesPageOK
-                                    },
-                                    teamPage: {
-                                        status: teamPageOK
-                                    }
-                                },
-                                database: {
-                                    status: sqlConnectionOK
-                                }
+
+            var query = "SELECT 1;"
+            connection.query(query, function (error, results, fields) {
+                sqlConnectionOK = !error;
```

Comment:
What is !error doing?  Want to make that wasn't supposed to be !!error to get it's logical state and not reverse it.  I could be misreading this of course.

Reply:
this is saying "if `error` is falsey, `sqlConnectionOK` will be `true`", which is the correct logic

Reply:
gotcha 👍